src/note.ts

```diff
@@ -599,4 +599,34 @@ export abstract class Note extends Tickable {
   setPreFormatted(value: boolean): void {
     this.preFormatted = value;
   }
+
+  // Get the direction of the stem
+  getStemDirection(): number {
```

Comment:
This is causing a merge conflict because it was also in Migration/tremolo. You can remove it.

Reply:
done